### PR TITLE
Kc small updates

### PR DESCRIPTION
**Pull request by kev1n80**
+ Add smooth scrolling to new listing page when the user clicks on preview button
+ Left align listing information to make it easier for user's to read
Screenshot 2020-08-20 at 10 35 11 AM
+ Remove keyboard functionality for the navbar, since this is geared more for mobile users
+ In user page, tabs only show outline when tabbed on, but not when clicked on.
  - Clicked on
Screenshot 2020-08-20 at 10 36 37 AM
  - Tabbed on
Screenshot 2020-08-20 at 10 36 45 AM

### CODE REVIEW

File: src/main/webapp/scripts/user.js [hunk 1]
```diff
@@ -182,21 +182,33 @@ function createListingTabs(listingsDisplay, createdListingsId,
  * @param elementId the id of the element associated with this tab
  * @param hNum the number for the heading (ex: h1 ,h2, h3)
  * @param tabName the name of this name that is displayed to the user
+ * @param isDefaultTab if this is the defailt tab, highlight it
```
Comment (zrideaux) [inline src/main/webapp/scripts/user.js:185]: Typo here, should be default

File: src/main/webapp/scripts/user.js [hunk 2]
```diff
@@ -182,21 +182,33 @@ function createListingTabs(listingsDisplay, createdListingsId,
...
+  const hTabContent = createHElement(tabName, hNum, 'tab-content pill ' + tabContentClass,
+      tabId);
+  hTab.appendChild(hTabContent);
+
+  hTabContent.setAttribute("tabindex", "-1");
+
+  // When h elemnt is clicked on, change the display to elementDisplay.
+  hTabContent.addEventListener("click", function(){ 
     toggleTabDisplay(elementDisplay, elementId, elementOtherId, otherTabId,   
         tabId)
```
Comment (zrideaux) [inline src/main/webapp/scripts/user.js:208]: Add a semicolon at the end of line 208 and a space between function() and the open brace on line 206

File: src/main/webapp/stylesheets/user.css [hunk 3]
```diff
@@ -8,14 +8,17 @@ a {
...
   margin-right: 1.5rem;
 }
 
 .profile {
   align-self: flex-start;
 }
 
+.tab-default {
+  background: #8EEEDE;
+}
```
Comment (zrideaux) [inline src/main/webapp/stylesheets/user.css:20]: Would a better name for this class be something like `.tab-selected`/`.selected-tab`? Then a followup could be to assign it to whatever tab was clicked in `toggleTabDisplay`. Just a thought!

File: src/main/webapp/scripts/user.js (hunk 1 above)
Comment (kev1n80) [reply src/main/webapp/scripts/user.js:185]: Yup, nice catch!

File: src/main/webapp/stylesheets/user.css (hunk 3 above)
Comment (kev1n80) [reply src/main/webapp/stylesheets/user.css:20]: Great idea! I will make these changes.

**Pull request merged**